var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -167,6 +170,12 @@ def cmake_args(self):
                     '-DModule_vtkGUISupportQtOpenGL:BOOL=ON',
                 ])
 
+        if '+mpi' in spec:
+            cmake_args.extend([
+                '-DVTK_Group_MPI:BOOL=ON',
+                '-DVTK_USE_SYSTEM_DIY2=OFF'
+            ])
+                
```

Comment:
flake8 error on this line (trailing whitespaces)
https://travis-ci.org/spack/spack/jobs/440071587